### ISSUE TITLE
Skill queue alignment issue

### ISSUE THREAD

**Issue by Rakdos8** [eveseat/seat#571]
* **Problem**: 
The ordered list is malformed. Probably a CSS or HTML issue due to line break.
Maybe a bootstrat issue ?

* **Expected**: 
The ordered list should be before the skill name

* **Logs / Screenshots / Proof**: 
![image](https://user-images.githubusercontent.com/22244611/61731708-51b6df80-ad7c-11e9-9b22-1ebcadad42bb.png)

* **Version Info**: 
OS:
`Debian 9.4 64b`

PHP: 
```txt
PHP 7.2.20-1+0~20190710.23+debian9~1.gbp2428c5 (cli) (built: Jul 10 2019 07:46:03) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.2.0, Copyright (c) 1998-2018 Zend Technologies
    with Zend OPcache v7.2.20-1+0~20190710.23+debian9~1.gbp2428c5, Copyright (c) 1999-2018, by Zend Technologies
```

SeAT versions:
```
| Vendor  | Package Name  | Installed Version |
| ------- | ------------- | ----------------- |
| eveseat | api           | 3.0.7             |
| eveseat | console       | 3.0.5             |
| eveseat | eveapi        | 3.0.15            |
| eveseat | notifications | 3.0.5             |
| eveseat | services      | 3.0.8             |
| eveseat | web           | 3.0.21            |
```

Web browser: Opera. Did not try with other

**Comment by warlof** [comment]
Hi @Rakdos8 what is your device resolutions?

**Comment by Rakdos8** [comment]
Hi @warlof !

Well, it's not a really known resolution but here we go: 2736x1824 with a zoom factor of 200%. Default resolution on a Microsoft Surface Pro 4.

Note that when a collapse the left menu, the ordrer list is printed correctly.
![image](https://user-images.githubusercontent.com/22244611/61799830-1d026100-ae2c-11e9-8933-869e613c6ff2.png)

Native layout:
![image](https://user-images.githubusercontent.com/22244611/61799872-33a8b800-ae2c-11e9-80a7-de6acf0e2a01.png)

**Comment by benthompson0** [comment]
Attempted to replicate this issue in SeAT 4.x on a Surface Pro (similar resolution, at 200% display scaling) - could not reproduce. Looks like it was fixed by eveseat/web#390.

@warlof recommend closing this issue.
